cmd/bnscli/clitests/attach_multisig_id.test

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+bnscli multisig -activation 4 -admin 8 \
+	| bnscli with-multisig 1 2 \
+	| bnscli with-multisig 3 \
+	| bnscli view
```

Comment:
Funny example that it creates a multi-sig with no participants?
Hmmm... maybe a better example, using those `with-mutlisig` on a send command?